proxy/logging/Makefile.am

```diff
@@ -80,7 +80,6 @@ test_LogUtils_CPPFLAGS = \
 
 test_LogUtils_SOURCES = \
 	LogUtils.cc \
-	unit-tests/BufferWriterFormat.cc \
```

Comment:
Can we delete the file too? It looks like it's only used here.

Reply:
Ah right.